core/include/detray/navigation/detail/helix.hpp

```diff
@@ -67,11 +67,11 @@ class helix {
         _h0 = vector::normalize(*_mag_field);
 
         // Normalized tangent vector
-        _t0 = vector::normalize(dir);
+        _t0 = dir;
```

Comment:
We can put `assert(abs(getter::norm(_t0) - 1) < 1e-5)`  instead

Reply:
Done